core/src/test/kotlin/com/segment/analytics/kotlin/core/PluginTests.kt

```diff
@@ -36,7 +36,7 @@ class PluginTests {
     }
 
     @Test
-    fun `execute runs the destination timeline`() {
+    fun `plugin processes the event correctly`() {
```

Comment:
nothing to do with this change, i just realized that the name was incorrectly written